tools/wptrunner/wptrunner/testloader.py

```diff
@@ -546,9 +546,8 @@ def group_metadata(cls, state):
     def group(self):
         if not self.current_group.group or len(self.current_group.group) == 0:
             try:
-                self.current_group = self.test_queue.get(block=True, timeout=5)
+                self.current_group = self.test_queue.get_nowait()
             except Empty:
-                self.logger.warning("Timed out getting test group from queue")
```

Comment:
Keep this line of log might be useful. We are not supposed to reach this line due to the existence of sentinal.

If we are going to make a change at testrunner side, we won't need a change here.

Reply:
I agree that we should keep a warning here though.